kustomize/overlays/prod/raw/config.toml

```diff
@@ -17,6 +17,7 @@ template_file = "message.tmpl"
 [app.http_client]
 max_idle_conns =  100
 request_timeout = 8000
+proxy_url = "http://115.124.92.42:8080"
```

Comment:
You should redact this.

Reply:
Oh sorry, that was still from my test. I replaced it with the placeholder, as I did with the other examples.